### DISCUSSION TITLE
layerdiffuse problem. how to fix this?

### DISCUSSION THREAD

**Discussion by rafiislambd** [Panchovix/stable-diffusion-webui-reForge#28] [Q&A] [answered]
*** Arguments: ('task(xemhh21ds9pvhkn)', <gradio.routes.Request object at 0x000001903D67F610>, 'red Ribbon Knot Ribbon Bow, ', '', [], 1, 1, 1.5, 1042, 1024, False, 0.7, 2, 'Latent', 0, 0, 0, 'Use same checkpoint', 'Use same sampler', 'Use same scheduler', '', '', [], 0, 8, 'DPM++ 2M SDE', 'SGM Uniform', False, '', 0.8, -1, False, -1, 0, 0, 0, False, False, False, False, 'base', True, '(SDXL) Only Generate Transparent Image (Attention Injection)', 1, 1, None, None, None, 'Crop and Resize', False, '', '', '', ControlNetUnit(input_mode=<InputMode.SIMPLE: 'simple'>, use_preview_as_input=False, batch_image_dir='', batch_mask_dir='', batch_input_gallery=[], batch_mask_gallery=[], multi_inputs_gallery=[], generated_image=None, mask_image=None, hr_option=<HiResFixOption.BOTH: 'Both'>, enabled=False, module='None', model='None', weight=1, image=None, resize_mode=<ResizeMode.INNER_FIT: 'Crop and Resize'>, processor_res=-1, threshold_a=-1, threshold_b=-1, guidance_start=0, guidance_end=1, pixel_perfect=False, control_mode=<ControlMode.BALANCED: 'Balanced'>, save_detected_map=True), ControlNetUnit(input_mode=<InputMode.SIMPLE: 'simple'>, use_preview_as_input=False, batch_image_dir='', batch_mask_dir='', batch_input_gallery=[], batch_mask_gallery=[], multi_inputs_gallery=[], generated_image=None, mask_image=None, hr_option=<HiResFixOption.BOTH: 'Both'>, enabled=False, module='None', model='None', weight=1, image=None, resize_mode=<ResizeMode.INNER_FIT: 'Crop and Resize'>, processor_res=-1, threshold_a=-1, threshold_b=-1, guidance_start=0, guidance_end=1, pixel_perfect=False, control_mode=<ControlMode.BALANCED: 'Balanced'>, save_detected_map=True), ControlNetUnit(input_mode=<InputMode.SIMPLE: 'simple'>, use_preview_as_input=False, batch_image_dir='', batch_mask_dir='', batch_input_gallery=[], batch_mask_gallery=[], multi_inputs_gallery=[], generated_image=None, mask_image=None, hr_option=<HiResFixOption.BOTH: 'Both'>, enabled=False, module='None', model='None', weight=1, image=None, resize_mode=<ResizeMode.INNER_FIT: 'Crop and Resize'>, processor_res=-1, threshold_a=-1, threshold_b=-1, guidance_start=0, guidance_end=1, pixel_perfect=False, control_mode=<ControlMode.BALANCED: 'Balanced'>, save_detected_map=True), False, 7, 1, 'Constant', 0, 'Constant', 0, 1, 'enable', 'MEAN', 'AD', 1, False, 1.01, 1.02, 0.99, 0.95, False, 0.5, 2, False, 256, 2, 0, False, False, 3, 2, 0, 0.35, True, 'bicubic', 'bicubic', False, 0, 'anisotropic', 0, 'reinhard', 100, 0, 'subtract', 0, 0, 'gaussian', 'add', 0, 100, 127, 0, 'hard_clamp', 5, 0, 'None', 'None', False, 'MultiDiffusion', 768, 768, 64, 4, False, False, False, False, False, 'positive', 'comma', 0, False, False, 'start', '', 1, '', [], 0, '', [], 0, '', [], True, False, False, False, False, False, False, 0, False) {}
    Traceback (most recent call last):
      File "E:\Ai Softwear\stable-diffusion-webui-reForge\modules\call_queue.py", line 74, in f
        res = list(func(*args, **kwargs))
    TypeError: 'NoneType' object is not iterable

**Comment by Panchovix** [comment]
Can you give me the complete problem shown in the console?

**Reply by rafiislambd** [reply]
![555+](https://github.com/user-attachments/assets/abb7af8d-f12a-419b-8a1d-72b434bdd7f9)
![555](https://github.com/user-attachments/assets/3e9c9716-6155-4184-861d-e6747b4253d5)

**Reply by rafiislambd** [reply]
already update but still same problem but original forge working fine

**Reply by Panchovix** [reply]
Update to latest commit 
![imagen](https://github.com/user-attachments/assets/c249fede-9237-446b-ba21-fd1c64731f6b)

**Reply by rafiislambd** [reply]
working .. thank you so much

**Comment by Panchovix** [comment, marked as answer]
I have fixed the issue in latest commit.

https://github.com/Panchovix/stable-diffusion-webui-reForge/commit/eea0d9b9d791bdbbef8a7bd0d82d4b1e349ecb7b

Thanks for the report.
